tests/bettertransformer/test_bettertransformer_vision.py

```diff
@@ -30,6 +30,11 @@
 ]
 
 
+ALL_VISION_TEXT_MODELS_TO_TEST = [
+    "hf-internal-testing/tiny-random-ViltModel",
```

Comment:
```suggestion
    "hf-internal-testing/tiny-vilt-random-vqa",
```
I think the previous model is broken, let's try with this one!

Reply:
Thanks for the feedback. I did fix the alphabetic order and added the test for vision and text models, but the test still didn't pass.

Reply:
Yeah I saw that! Thanks a lot for fixing the alphabetical order ;) ! let's see now if the test passes with the new model